datacats/cli/manage.py

```diff
@@ -66,15 +67,16 @@ def reload_(environment, opts):
     """Reload environment source and configuration
 
 Usage:
-  datacats reload [-bp] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT [PORT]]
+  datacats reload [-b] [-p|--no-reload] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT [PORT]]
```

Comment:
I'm thinking either:
1. `datacats restart` while quietly redirecting `reload` to the new command.
2. `datacats reload --no-watch` to say that we're not watching the files for changes